framework/test-vectors/complete-vectors/hierarchy.md

```diff
@@ -21,3 +21,54 @@ with every available [algorithm suite](../../algorithm-suites.md#algorithm-suite
 
 For every available algorithm suite and static branch key,
 a test MUST attempt to encrypt and decrypt with every [standard encryption context](./encryption-context.md#standard-encryption-contexts).
+
+### Input dimensions
+
+- encrypt key description.key: range of [representative branch keys](#representative-branch-keys)
+- decrypt key description.key: range of [representative branch keys](#representative-branch-keys)
+- decrypt key description.logicalKeyStore:
+  - "Default",
+    - Represents the logical key store name on encrypt
+  - "Other"
+    - Represents any other logical key store name
+
+### Representative branch keys
+
+- `"static-branch-key-1"`
+  - Any valid branch key.
+- `"static-branch-key-2"`
+  - Any other valid branch key
+- `"branch-key-no-permissions"`
+  - Any valid branch key where
+    the test vector runner does not have permissions
+    for the KMS key
+- `"branch-key-not-in-table"`
+  - Any branch key ID not in the keystore table
+- `"branch-key-no-version"`
+  - Any branch key without a version
```

Comment:
Issue/Suggestion: "branch-key-no-version" is a little vague to me.
I read this as a Branch Key Record that was missing information.
But I think the intention is to make a request for a Version that does not exist.
Otherwise, this identical to "invalid-branch-key".
```suggestion
- `"branch-key-no-version"`
  - A Version of a branch key which does not exist
```